src/block.ts

```diff
@@ -245,16 +255,19 @@ const imageStyles = css`
     }
 `;
 
-const Image = ({ url, alt, salt, caption, displayCredit, credit }: ImageProps): ReactElement =>
-    styledH('figure', { css: imageStyles },
+const Image = (imageProps: ImageProps): ReactElement => {
+    const { url, alt, salt, caption, displayCredit, credit, typeData } = imageProps;
```

Comment:
Minor point, but the [React convention](https://reactjs.org/docs/components-and-props.html) is to name the parameter `props`.